README.md

```diff
@@ -10,7 +10,7 @@ If you like what you see and have a project you need coded, don't hesitate to co
 
```

Comment:
- [ ] The _dervServer_ package is missing your application. Kindly install the same and add that in your **webpack** configuration, by running the command  `npm install -D webpack-dev-server`
to enable your application to be run on the server locally.

**devServer in webpack configuration file:**
![image](https://user-images.githubusercontent.com/12745474/134922749-d0016f56-002e-4415-8882-4eb3dcbf6551.png)